app/models/hiring.rb

```diff
@@ -9,7 +9,8 @@ class Hiring < ApplicationRecord
   delegate :branch_name, to: :vehicle
   delegate :name, to: :user, prefix: :user
 
-  validates :time, presence: true
+  validates :time, presence: true, numericality: {greater_than_or_equal_to: Setting.min_time,
+    only_integer: true}
```

Comment:
đọc lại cái coding convention này sắp xếp lại cho c thứ tự các block xem nào
https://github.com/framgia/coding-standards/blob/master/eng/rails/standard.md
search chữ Scope should be written in short style like lambda. If there are more than 80 characters in a line, it should be cut to a new line so there are less than 80 characters in a line.

Reply:
E sửa thế này đc đúngchưa ạ